core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -271,39 +277,45 @@ trait Endpoint[F[_], A] { self =>
     *
     * Consider using [[io.finch.Bootstrap]] instead.
     */
-  final def toService(implicit
-    tr: ToResponse.Aux[A, Application.Json],
-    tre: ToResponse.Aux[Exception, Application.Json],
-    F: Effect[F]
-  ): Service[Request, Response] = toServiceAs[Application.Json]
+  final def toService(
+      implicit
+      tr: ToResponse.Aux[A, Application.Json],
+      tre: ToResponse.Aux[Exception, Application.Json],
+      F: Effect[F]
+    ): Service[Request, Response] = toServiceAs[Application.Json]
 
   /**
     * Converts this endpoint to a Finagle service `Request => Future[Response]` that serves custom
     * content-type `CT`.
     *
     * Consider using [[io.finch.Bootstrap]] instead.
     */
-  final def toServiceAs[CT <: String](implicit
-    tr: ToResponse.Aux[A, CT],
-    tre: ToResponse.Aux[Exception, CT],
-    F: Effect[F]
-  ): Service[Request, Response] = Bootstrap.serve[CT](this).toService
+  final def toServiceAs[CT <: String](
+      implicit
+      tr: ToResponse.Aux[A, CT],
+      tre: ToResponse.Aux[Exception, CT],
+      F: Effect[F]
+    ): Service[Request, Response] = Bootstrap.serve[CT](this).toService
 
   /**
     * Recovers from any exception occurred in this endpoint by creating a new endpoint that will
     * handle any matching throwable from the underlying future.
     */
-  final def rescue(pf: PartialFunction[Throwable, F[Output[A]]])(implicit
-    F: ApplicativeError[F, Throwable]
-  ): Endpoint[F, A] = transform(foa => foa.handleErrorWith(pf))
+  final def rescue(
```

Comment:
Is it just me or this is now much harder to grok?

Reply:
yeah I agree. I suspect this is related to `verticalMultiline`. I can try shutting it off and see how it looks.